frontend/src/components/pages/GlobalForms/index.tsx

```diff
@@ -78,6 +79,28 @@ const GlobalFormsPage = (): React.ReactElement => {
     }
   };
 
+  const saveFormQuestion = async (formQuestion: CreateFormQuestion) => {
+    const newFormQuestion = await AdminAPIClient.addQuestionToTemplate(
+      formQuestion,
+    );
+
+    if (newFormQuestion.id !== undefined) {
+      toast({
+        description: `Form question was added to the form template`,
+        status: "success",
+        variant: "subtle",
+        duration: 3000,
+      });
```

Comment:
We should make this a custom component called `SuccessToast` which takes in the description and maybe duration. You want me to do that in this ticket? I think we should do it in another PR since it's gonna hit pretty much every other page lol

Reply:
ehh not sure we can make it an actual component, but def out of scope of this ticket